src/voice-pe/using-the-device/new-ha-instance

```diff
@@ -0,0 +1,45 @@
+---
+zendesk:
+  article_id: 30670495779997
+  name: Connecting your Home Assistant Voice PE to a new Home Assistant Instance
+  position: 10
+  labels: voice pe, operation, using the device
+---
+
+When connecting your Home Assistant Voice Preview Edition (VPE) device to a new Home Assistant instance, a factory reset is required. This is because the VPE device stores an encryption key tied to the original Home Assistant instance, even if the device is removed from that setup.
```

Comment:
```suggestion
When connecting your Home Assistant Voice Preview Edition device to a new Home Assistant instance, a factory reset is required. This is because the device stores an encryption key tied to the original Home Assistant instance, even if the device is removed from that setup.
```
I'm guilty of the crime myself, but I think we're not supposed to call it VPE.

Reply:
https://github.com/NabuCasa/support/pull/692